tests/table/utf8/test_length.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+
+from daft.expressions import col
+from daft.table import Table
+
+
+def test_utf8_length():
+    table = Table.from_pydict({"col": ["foo", None, "barbaz", "quux"]})
+    result = table.eval_expression_list([col("col").str.length()])
+    assert result.to_pydict() == {"col": [3, None, 6, 4]}
```

Comment:
So we don't really have a dev best-practice around this just yet, but how have you been thinking about when to write a Series test vs Table tests?

Seems like this is well-covered in your tests in `tests/series/test_utf8_ops.py` as well?

Reply:
Yeah I wanted to ask about that, I've been erring on the side of redundant coverage to ensure that we have at least one e2e test for table-level ops, even if the op just delegates to the series-level op. Normally I'd mock out the series-level op and have the table-level op test assert that we delegate to the series-level op as expected, but we've got a pesky Python <--> Rust language boundary there. So I decided to do a bare-minimum, single e2e test for applying the op to a table instead of a series, just to make sure the former works as expected and to guard against a future refactor that breaks the expression delegation.

I should also note - we do something similar for other tests, e.g. for the other string ops: https://github.com/Eventual-Inc/Daft/blob/a81eea8ffe89d6ba51620d74e4d5cb1304a74df7/tests/table/utf8/test_compares.py#L13-L30

Reply:
I don't feel too strongly about this, so if you feel like it's overly redundant, I'd be happy to delete it!

Reply:
We don't really have any strict standards around testing atm, so I say send it! 🤓